packages/babel-cli/src/babel/index.js

```diff
@@ -240,10 +240,6 @@ if (errors.length) {
 //
 
 const opts = commander.opts();
-//the configFile CLI option maps to the extends option in the node API
-if (opts.configFile) {
```

Comment:
didn't know this was an option before but I see from the comment it was just extends

Reply:
Yeah it landed back in https://github.com/babel/babel/pull/6133 but I've always wanted to kill it and make us have a proper config file instead :)